5m5v-bot.js

```diff
@@ -1,97 +1,115 @@
 #!/usr/bin/env node
 
-const fs = require('fs');
-const yaml = require('js-yaml');
-const { Rettiwt } = require('rettiwt-api');
-const TweetFilter = require('./lib/filter');
-const util = require('./lib/util');
+require('dotenv').config();
 
-let config = {};
-try {
-  config = yaml.safeLoad(fs.readFileSync('5m5v-config.yaml', 'utf8'));
-} catch (error) {
-  throw `Unable to load config file: ${error.message}`;
-}
+['TWEETS_API_ENDPOINT', 'TWEETS_API_KEY', 'TWITTER_EMAIL', 'TWITTER_USERNAME', 'TWITTER_PASSWORD'].forEach(key => {
+  if (!process.env[key]) {
+    console.error(`Environment variable ${key} is not set (or empty). Exiting.`);
+    process.exit(1);
+  }
+});
```

Comment:
Would probably replace that with envsyouwanttocheck.every and check inside if the key exists, that way you can prevent having that check inside the foreach